src/init.ts

```diff
@@ -75,8 +80,38 @@ async function init(targetDir: string, repository: string, options: any): Promis
 
 }
 
+async function initCellsTemplate(targetDir: string, cellSiteTemplateRepo: string) {
```

Comment:
Parameter `cellSiteTemplateRepo` overrides const `cellSiteTemplateRepo` declared at the beginning of the file.

Reply:
This comment is now obsolete.